main.lua

```diff
@@ -40,12 +79,17 @@ lib.load = function ()
   -- Add love-api to the module load path.
   package.path = package.path .. ';love-api/?.lua'
   local API = require("love_api")
-  printModule(lib, API, libTxt, "")
+  local results = checkModule(lib, API, libTxt, true)
+  printResults(results, 1)
 end
 
 lib.update = function (dt)
   -- Now that the application has loaded, exit.
   lib.window.close()
+  -- Note: closing the window does not exit love
+  if (lib.event ~= nil) and (lib.event.quit ~= nil) then
+    lib.event.quit()
```

Comment:
Also added `lib.event.quit()` to Lutro recently too. Should we should be good to switch to that once builds are out.